docs/aerodromes/classd/Bankstown.md

```diff
@@ -61,16 +61,16 @@ Circuits to be flown at `A010`
 
 ## Helicopter Operations
 ### General
-These procedures apply during hours of daylight only.  During hours of darkness, all helicopters must revert to fixed-wing operations.  
+These procedures apply during hours of daylight only. During hours of darkness, all helicopters must revert to fixed-wing operations.  
 
-The Main Pad (abeam taxiway Mike) is treated like a runway and requires a takeoff/landing clearance.  Helicopters are permitted to become airborne from a limited number of other locations on the aerodrome, such as taxiway November Two, and should be instructed to *"report airborne"* or *"report on the ground"*.
+The Main Pad (abeam taxiway Mike) is treated like a runway and requires a takeoff/landing clearance. Helicopters are permitted to become airborne from a limited number of other locations on the aerodrome, such as taxiway November Two, and should be instructed to *"report airborne"* or *"report on the ground"*.
 
 ### Reporting Points
-Three helicopter reporting points help keep helicopters segregated from other traffic.  
+Three helicopter reporting points help keep helicopters separated from other traffic.  
```

Comment:
The original wording is correct. The reporting points do not automatically separate with other traffic, they *help* separate, by *segregating* the routes

Reply:
Gotcha, no worries